cli/cmd/migrate_v3.go

```diff
@@ -40,7 +52,7 @@ func migrateConnectionV3(ctx context.Context, sourceClient *managedplugin.Client
 			opts = append(opts, transformer.WithRemovePKs())
 			opts = append(opts, transformer.WithCQIDPrimaryKey())
 		}
-		destinationTransformers[i] = transformer.NewRecordTransformer(opts...)
+		destinationRecordTransformers[i] = transformer.NewRecordTransformer(opts...)
```

Comment:
These are unrelated (we already had a concept of RecordTransformer that adds `_cq_*` fields. I renamed them to make it _less_ confusing.

Reply:
Can we remove the old record transformers and move them to the new transformers? Maybe we can have "default" transformers?

Reply:
Yes, I can do that. But that requires non-trivial changes in `sync_v3` too. I'd prefer to do that separately fi that's ok?

Reply:
(added issue https://github.com/cloudquery/cloudquery-issues/issues/2386)